react-example/src/components/Arkose.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Arkose extends React.Component {
+  constructor () {
+    super();
+    this.myEnforcement = null;
+  }
+
+  // Append the JS tag to the Document Body.
+  loadScript = () => {
+    const scriptId = 'arkose-script';
+    const currentScript = document.getElementById(scriptId);
+    if (currentScript) {
+      currentScript.remove();
+    }
+    const script = document.createElement('script');
+    script.id = scriptId;
+    script.type = 'text/javascript';
+    script.src = `https://client-api.arkoselabs.com/v2/${this.props.publicKey}/api.js`;
+    script.setAttribute('data-callback', 'setupEnforcement');
+    script.async = true;
+    script.defer = true;
+    if (this.props.nonce) {
+      script.setAttribute('data-nonce', this.props.nonce);
+    }
+    document.body.appendChild(script);
+    return script;
+  };
+
+  setupEnforcement = (myEnforcement) => {
+    this.myEnforcement = myEnforcement;
+    this.myEnforcement.setConfig({
+      selector: this.props.selector && `#${this.props.selector}`,
```

Comment:
do we convert to an id selector in the other examples? If so we should probably change that, because a selector doesn't need to be an id, it can be any valid DOM selector, also if someone passes in an id selector it'll append another `#`

Reply:
Good point. Yeah it's the same in other examples as well. So I'll update that

Reply:
I'll update the same in other examples in another PR